src/cudnn/batchnorm.jl

```diff
@@ -68,57 +84,82 @@ function cudnnBNForward!(y::DenseCuArray{T}, g::DenseCuArray{T}, b::DenseCuArray
   return y
 end
 
+function ∇batchnorm(g::Nothing, b::Nothing, x::DenseCuArray, dy::DenseCuArray,
+                  running_mean, running_var, momentum; kws...)
+  g = fill!(similar(x, size(ndims(x)-1)), 1)
+  b = fill!(similar(x, size(ndims(x)-1)), 0)
+  ∇batchnorm(g, b, x, dy, running_mean, running_var, momentum; kws...)
+end
+
 function ∇batchnorm(g::DenseCuArray{T}, b::DenseCuArray{T}, x::DenseCuArray{T, 2}, dy::DenseCuArray{T, 2},
```

Comment:
All of these signatures are probably too restrictive now that cuDNN supports mixed precision batchnorm. Not a problem for this PR though, let's revisit in a follow-up.